packages/python-google-compute-engine/google_compute_engine/networking/network_setup/network_setup.py

```diff
@@ -43,6 +44,22 @@ def __init__(self, dhclient_script=None, dhcp_command=None, debug=False):
         name='network-setup', debug=debug, facility=facility)
     self.distro_utils = distro_utils.Utils(debug=debug)
 
+  def EnableIpv6(self, interfaces):
+    """Enable IPv6 on the list of network interfaces.
+
+    Args:
+      interfaces: list of string, the output device names for enabling IPv6.
+    """
+    if not interfaces or set(interfaces) == self.ipv6_interfaces:
+      return
+
+    self.logger.info('Enabling IPv6 on Ethernet interface: %s.', interfaces)
```

Comment:
three references to this var, this is the only one not using the set() version. maybe interfaces = set(interfaces) as the first statement in this method?

Reply:
Done. Also updated the function below for consistency.